lib/sidekiq_tracer/sidekiq/middleware/client/tracer.rb

```diff
@@ -5,18 +5,8 @@ module Client
         class Tracer
           def call(worker_class, msg, queue, _redis_pool)
             msg['request_id'] ||= Current.web_request_id if defined?(Current) && Current.web_request_id.present?
-            dist_trace_id = SidekiqTracer.skip_distributed_trace_on&.call(worker_class, msg, queue) ? nil : msg['trace_id']
```

Comment:
@noxasch 

1. leave a hook in the config for ppl to do this, sth like `custom_client_distributed_tracer` and invoke it here.
2. add this as an example in the readme
3. bump the minor version of the gem